deployments/pulse3d/services/pulse3d/src/repository/notification_repository.py

```diff
@@ -50,3 +51,24 @@ async def get_all(self) -> list[NotificationResponse]:
             notifications = await con.fetch(query)
 
         return [NotificationResponse(**dict(notification)) for notification in notifications]
+
+    async def get_notification_messages(
+        self, account_id: str, notification_message_id: str
+    ) -> list[NotificationMessageResponse]:
+        query = f"""
+            SELECT m.id, m.created_at, m.viewed_at, n.subject, n.body
+            FROM notification_messages m, notifications n
+            WHERE m.recipient_id = '{account_id}'
+            AND m.notification_id = n.id
+        """
+
+        if notification_message_id:
+            query += f" AND m.id = '{notification_message_id}'"
```

Comment:
We might want to avoid using f strings here to avoid a SQL injection attack. Something like this should work
```
query = f"""
  SELECT m.id, m.created_at, m.viewed_at, n.subject, n.body
  FROM notification_messages m, notifications n
  WHERE m.recipient_id = $1
   AND m.notification_id = n.id
"""
query_params = [account_id]
if notification_message_id:
  query += f" AND m.id = $2"
  query_params.append(notification_message_id)
...
await con.fetch(query, *query_params)
```